test/distributed/test_pg_wrapper.py

```diff
@@ -65,59 +94,88 @@ def _test_collectives_op_mismatch(self, wrapper_pg, use_cuda=False):
             w.wait()
 
         # Simulate mismatch: allreduce vs reduce.
-        with self.assertRaisesRegex(
-            RuntimeError, "Mismatch between collective operation types"
-        ):
+        # Error including info about inconsistent collective, rank, tensor
+        # shape, device, and dtype should be raised.
+        with self.assertRaisesRegex(RuntimeError, "..") as cm:
```

Comment:
What exactly do we match by `..` in this (and the following) regular expressions?

Reply:
The regex is just intended to match any character, because we are only interested in ensuring that we raise a `RuntimeError` and then check the actual error message below. I guess we could change the regex to ".*" to be more clear or just make it self.assertRaises.